src/mame/taito/taito_f3_v.cpp

```diff
@@ -1258,33 +1262,43 @@ void taito_f3_state::init_alpha_blend_func()
 
 /******************************************************************************/
 
-#define GET_PIXMAP_POINTER(pf_num) \
-{ \
-	const f3_playfield_line_inf *line_tmp = line_t[pf_num]; \
-	m_src[pf_num] = line_tmp->src[y]; \
-	m_src_s[pf_num] = line_tmp->src_s[y]; \
-	m_src_e[pf_num] = line_tmp->src_e[y]; \
-	m_tsrc[pf_num] = line_tmp->tsrc[y]; \
-	m_tsrc_s[pf_num] = line_tmp->tsrc_s[y]; \
-	m_x_count[pf_num] = line_tmp->x_count[y]; \
-	m_x_zoom[pf_num] = line_tmp->x_zoom[y]; \
-	m_clip_al[pf_num] = line_tmp->clip_in[y] & 0xffff; \
-	m_clip_ar[pf_num] = line_tmp->clip_in[y] >> 16; \
-	m_clip_bl[pf_num] = line_tmp->clip_ex[y] & 0xffff; \
-	m_clip_br[pf_num] = line_tmp->clip_ex[y] >> 16; \
-	m_pal_add[pf_num] = line_tmp->pal_add[y]; \
+void taito_f3_state::get_pixmap_pointer(const int skip_layer_num, const f3_playfield_line_inf **line_t, const int y)
+{
+	for(int pf_num = skip_layer_num; pf_num < 5; ++pf_num)
+	{
```

Comment:
Please be consistent with spacing – the rest of the file has a space between the flow control keyword and the opening parenthesis of the controlling expression.